ovirt/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+docker build ovirt/ -t localhost:5001/fakeovirt
```

Comment:
podman? same for line below

Reply:
The kind run is running in docker and would not see podman images